core/config/locales/en.yml

```diff
@@ -865,6 +864,7 @@ en:
           single_code_html: 'This promotion uses the promotion code: <code>%{code}</code>'
         activations_new:
           auto: Apply to all orders
+          auto_warning: Promo codes will be disabled
```

Comment:
This message is not very informative, we should expand a little bit more.

Reply:
I chose a short text because of an error in UI with long texts:
<img width="1676" alt="Schermata 2020-02-06 alle 11 48 50" src="https://user-images.githubusercontent.com/1719696/73930632-225e5f00-48d7-11ea-8263-34c73d7049ef.png">

Can I modify the layout to enlarge that col? Or maybe I can modify the pill's css to disable white-space nowrap

Reply:
I think we need this component there:

<img width="84" alt="Screenshot 2020-02-06 at 11 59 17" src="https://user-images.githubusercontent.com/167946/73931186-263eb100-48d8-11ea-91a5-cdc7a73fd7a3.png">

It should be called `hint`, my fault!